docs/faq.md

```diff
@@ -28,6 +28,21 @@ For detail information of each config field, please run:
 ```bash
 spicetify --help config
 ```
+## Spotify 'blank' after update
```

Comment:
I'd say that these issues occur to both platforms and should not have different steps for each.

Additionally "Ask around in the Spicetify discord server, there are lots of active GNU/Linux and macOS users ready to give you pointers." should not be a bullet point. Documentation gives you information on how to solve things, it is assumed that if you can't through documentation, the last resort should be to ask for support.